mautic/mautic
# Fix segment membership filter build query

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | 🟥 
| Deprecations?                          | 🟥 
| BC breaks? (use the c.x branch)        | 🟥 
| Automated tests included?              | ✅ 
| Related user documentation PR URL      | NA
| Related developer documentation PR URL | NA
| Issue(s) addressed                     | 



#### Description:

Query for include segment was build wrong. I fixed it and cover by unit tests



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create some segment with include/exclude segment membership 
3. Check If works like before




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10809"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
